apolloschurchapp/src/theme/theme.js

```diff
@@ -142,6 +143,14 @@ const overrides = {
     // eslint-disable-next-line react/display-name
     HeaderComponent: () => (props) => <ContentNodeHeader {...props} />,
   },
+  'ui-kit.DefaultCard': () => ({ relatedNode, actionIcon, labelText }) => ({
+    LabelComponent:
+      actionIcon || labelText || !relatedNode?.id ? (
```

Comment:
essentially if there is already an `actionIcon` or `labelText` we for sure know that we don't want to render the organization feature. If there's not, we'll attempt to load an organization feature, and for anything that doesn't have an org feature, that component will handle rendering `null`, so this doesn't impact every card.